bigchaindb/commands/bigchaindb.py

```diff
@@ -208,16 +222,35 @@ def create_parser():
                                help='The backend to use. It can only be '
                                '"localmongodb", currently.')
 
+    # parser for managing validator elections
     validator_parser = subparsers.add_parser('upsert-validator',
-                                             help='Add/update/delete a validator')
+                                             help='Add/update/delete a validator.')
+
+    validator_subparser = validator_parser.add_subparsers(title='Action',
+                                                          dest='action')
+
+    new_election_parser = validator_subparser.add_parser('new',
```

Comment:
upsert already implies add/update/delete, should we mention `new`?

Reply:
I don't understand the question. `new` refers to a new election, not a new validator. Is this not clear from the help message/parser name?

Reply:
I see, didn't check the BEP before. looks good.